bindings/python/README.md

```diff
@@ -86,21 +86,21 @@ that connects to the [Shimmer Testnet](https://api.testnet.shimmer.network) usin
 to safely store a seed derived from a mnemonic, and then print the account's information.
 
 ```python
-from iota_sdk import Wallet, StrongholdSecretManager, CoinType
+from iota_sdk import Wallet, StrongholdSecretManager, CoinType, ClientOptions
 
 # This example creates a new database and account
 
-wallet_options = {
-    'nodes': ['https://api.testnet.shimmer.network'],
-}
+client_options = ClientOptions(nodes=['https://api.testnet.shimmer.network'])
 
-secret_manager = StrongholdSecretManager("wallet.stronghold", "some_hopefully_secure_password")
+secret_manager = StrongholdSecretManager(
+    "wallet.stronghold", "some_hopefully_secure_password")
 
-wallet = Wallet('./alice-walletdb', wallet_options, coin_type=CoinType.SHIMMER, secret_manager)
+wallet = Wallet('./alice-walletdb', client_options,
+                CoinType.SHIMMER, secret_manager)
 
 # Store the mnemonic in the Stronghold snapshot. This only needs to be done once
 account = wallet.store_mnemonic("flame fever pig forward exact dash body idea link scrub tennis minute " +
-          "surge unaware prosper over waste kitten ceiling human knife arch situate civil")
+                                "surge unaware prosper over waste kitten ceiling human knife arch situate civil")
```

Comment:
not sure bout this

Reply:
Idk, I guess it was done in develop 🤷🏻‍♂️